.env.example

```
@@ -0,0 +1,16 @@
+# Backend app metadata
+APP_NAME=Shelfy API
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)